plexapi/media.py

```diff
@@ -234,27 +261,43 @@ class VideoStream(MediaPartStream):
     def _loadData(self, data):
         """ Load attribute values from Plex XML response. """
         super(VideoStream, self)._loadData(data)
+        self.anamorphic = data.attrib.get('anamorphic')
         self.bitDepth = cast(int, data.attrib.get('bitDepth'))
-        self.bitrate = cast(int, data.attrib.get('bitrate'))
         self.cabac = cast(int, data.attrib.get('cabac'))
+        self.chromaLocation = data.attrib.get('chromaLocation')
+        self.codedHeight = data.attrib.get('codedHeight')
+        self.codedWidth = data.attrib.get('codedWidth')
+        self.colorPrimaries = data.attrib.get('colorPrimaries')
+        self.colorRange = data.attrib.get('colorRange')
         self.chromaSubsampling = data.attrib.get('chromaSubsampling')
```

Comment:
`chromaSubsampling` alphabetical order.

Reply:
needs alphabetical order in a couple places